inspirehep/config.py

```diff
@@ -1149,6 +1149,10 @@ def _(x):
     }
 }
 
+# Legacy PID provider
+# ===================
+LEGACY_PID_PROVIDER = None # e.g. "http://server-name/batchuploader/allocaterecord"
```

Comment:
Please replace `example.org` instead of `server-name`, since this is the default recommended host to use in examples.